logistics_planning_base/views/logistics_schedule_views.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="logistics_schedule_view_tree" model="ir.ui.view">
+        <field name="name">logistics.schedule.tree</field>
+        <field name="model">logistics.schedule</field>
+        <field name="arch" type="xml">
+            <tree editable="top">
```

Comment:
Start with tree decoration... `cancel` schedules should be grayed, `done` in green and `draft` in red